src/theme/dialog/dialog.js

```diff
@@ -8,22 +8,21 @@ import Description from './description';
 import Options from './options';
 import styles from './dialog.css';
 
-const Dialog = ({
-  children, hasClose, hasBack, className, history,
-}) => {
+const Dialog = ({ children, hasClose, hasBack, className, history, size }) => {
   const onCloseClick = () => removeSearchParamsFromUrl(history, ['modal'], true);
   const onBackClick = () => history.goBack();
 
   return (
-    <div className={`${styles.wrapper} ${className}`}>
+    <div className={`${styles.wrapper} ${className} ${size ? styles[size] : ''}`}>
```

Comment:
if className and size are undefined it will look like this in the dom:
`class="wrapper  ">`

Excellent case for classNames lib to clean the dom and simplify the condition
`className={classNames(styles.wrapper, className, size && styles[size])}`

Now if they are empty, dom will look like:
`class="wrapper">`

Reply:
yhea true we could easily write one or use a package like `classNames` that simply performs this but I think that would be going out of scope for now for this PR

Reply:
Just wanted to share that we actually have classnames installed now :D